### ISSUE TITLE
Cannot find name 'LocalForage'.

### ISSUE THREAD

**Issue by PhillipZada** [driftyco/ionic-storage#74]
Since updating the package getting this error

## Typescript Error
Cannot find name 'LocalForage'.
```
.../node_modules/@ionic/storage/es2015/storage.d.ts
undefined
undefined
```

> "@ionic/storage": "^1.1.8"
Ionic Framework: ^2.0.0
Ionic Native: ^2.5.1
Ionic App Scripts: 1.0.1
Angular Core: ^2.4.6
Angular Compiler CLI: ^2.4.6
Node: 6.3.0
OS Platform: Windows 10
Navigator Platform: Win32
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36

**Comment by zarko-tg** [comment]
This is an issue that came with 1.1.8 and only on Windows OS as far as I can tell.

**Comment by chrisworrell** [comment]
kinda wondering when they will respond to this, i have tried everything to get it working, im stuck on 1.1.7.  i've tried a fresh wipe of my node_modules and reinstall and it appears there is an issue between the dependencies with 1.1.8 and the localforage version, what issue that is though is outside of my ability to see easily

**Comment by PhillipZada** [comment]
No idea, but this is a breaking change so I don't know why they haven't responded at least 😢

**Comment by Flink91** [comment]
Same problem with 1.1.8. I had to downgrade as well for now.
`npm install --save @ionic/storage@1.1.7`

**Comment by jsayol** [comment]
I've been doing some digging and I'm pretty sure the 1.1.8 version was generated using the `--noResolve` TS option. Because of this, the generated `storage.d.ts` file lacks the reference to the LocalForage types that was present in previous versions:
```
/// <reference types="localforage" />
```

The code here on github doesn't match the one published on npm though, so I can't really tell for sure nor create a PR to fix it.

cc @mlynch @danbucholtz

**Comment by mlynch** [comment]
Thanks for the heads up. @danbucholtz seems related to your 1.1.8 release?

**Comment by danbucholtz** [comment]
I'll check it out on Windows. It's working great for me on Mac with AoT and non-AoT builds.

Thanks,
Dan

**Comment by danbucholtz** [comment]
This is fixed in `1.1.9`. It seems like the issue was because we had a `^` dependency on TypeScript which changed the way `.d.ts` files are published. Hard-coding TypeScript 2.0.x fixed the issue for me.

Thanks,
Dan

**Comment by longzheng** [comment]
This is not fixed for me in 1.1.9

I still see ```/// <reference types="localforage" />``` in my ```storage.d.ts```

My ```package.json``` has
```
    "@ionic/storage": "1.1.9",
```
